docs/dependabot-for-private-packagist.md

```diff
@@ -0,0 +1,58 @@
+# Set up Dependabot with Private Packagist
+##
+
+[Dependabot](https://docs.github.com/en/code-security/getting-started/dependabot-quickstart-guide) informs you about vulnerabilities in the dependencies that you use in your repository and can automatically raise pull requests to keep your dependencies up-to-date.  
+
+This guide explains how to configure and set up Dependabot if you want to use it for packages hosted on [Private Packagist](https://packagist.com) step by step.  
+
+
+## Enable Dependabot in GitHub
+
+Start in your GitHub repository and go to “Settings”. In the “Security” section of the sidebar go to “Code security and analysis”, where you can enable Dependabot. In the process, you will create a dependabot.yaml [as shown as below](#dependabotyaml). Follow [this guide on GitHub](https://docs.github.com/en/code-security/getting-started/dependabot-quickstart-guide#enabling-dependabot-for-your-repository) for the complete steps to enable Dependabot.
+
+![enable dependabot](/Resources/public/img/docs/articles/dependabot-secret.png)
+
+When you click on configure, the dependabot.yaml will be created for you in the .github folder.
+
+### dependabot.yaml
+
+A minimal Dependabot config file would look like this:
+
+```yaml
+version: 2
+updates:
+  - package-ecosystem: "composer"
+    directory: "/"
+    registries:
+      - private_packagist
+    schedule:
+      interval: "weekly"
+registries:
+  private_packagist:
+    type: composer-repository
+    url: https://repo.packagist.com/acme/
+    username: token
+    password: ${{secrets.PRIVATE_PACKAGIST_PASSWORD}}
+```
+
+Replace the Composer URL `https://repo.packagist.com/acme/` with your own organization's Composer URL on packagist.com. In the example, the repository URL is for the organization on Private Packagist Cloud with the name `acme`.
+
+To grant Dependabot access to the packages on Private Packagist, you need to provide the secret `PRIVATE_PACKAGIST_PASSWORD` in your GitHub repository. In Settings under the section “Security,” there is “Secrets and Variables” for Dependabot.
+
+![Dependabot Secrets](/Resources/public/img/docs/articles/dependabot-secret.png)
+
+Create a Team Authentication Token under “Settings” and “Authentication Tokens” on Private Packagist. Copy the secret into the GitHub form.
```

Comment:
This should probably explain if this is update or read-only

Reply:
```suggestion
Create a Team Authentication Token under “Settings” and “Authentication Tokens” on Private Packagist with Update Access. Copy the secret into the GitHub form.
```

Reply:
like this?

Reply:
Sounds good!